src/libs/CompanyCardUtils.ts

```diff
@@ -0,0 +1,39 @@
+import type {ValueOf} from 'type-fest';
+import CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
+
+/**
+ * @param feedProvider
+ * @returns Title based on the feedProvider
+ */
+function getTranslationKeyForFeedDetails(feedProvider: ValueOf<typeof CONST.COMPANY_CARDS.CARD_TYPE> | undefined): TranslationPaths | '' {
```

Comment:
I don't think we need neither this nor the method below. Instead we can just do:
```
`workspace.companyCards.addNewCard.feedDetails.${feedProvider}.title`
```